apps/console/src/features/core/api/bulk-operations.ts

```diff
@@ -53,10 +76,28 @@ export const updateResources = (data: BulkDataPropsInterface): Promise<any> => {
     };
 
     return httpClient(requestConfig)
-        .then((response) => {
+        .then((response: HttpResponse) => {
+            const bulkResponse: BulkResponseInterface | undefined = response?.data;
+            let isSuccessful: boolean = true;
+
+            if (bulkResponse?.schemas?.includes("urn:ietf:params:scim:api:messages:2.0:BulkResponse")) {
+                bulkResponse?.Operations?.some((operation: BulkResponseOperationInterface) => {
+                    // Check if status code exists and if it is not 200.
+                    if (operation?.status?.code !== undefined && operation.status.code !== 200) {
```

Comment:
```suggestion
                    if (operation?.status?.code !== 200) {
```

Reply:
As I tested, those are not equal. I can use 
`!isNaN(operation?.status?.code) && operation.status.code !== 200` though